cypress/e2e/1-getting-started/todo.cy.js

```diff
@@ -0,0 +1,143 @@
+/// <reference types="cypress" />
```

Comment:
this file should not be part of the PR, the following paths should be included in the .gitignore file, as explained in [step 3.4:](https://github.com/testing-community/cypress-training-doc#3-instalaci%C3%B3n-de-cypress)
```
cypress/e2e/1-getting-started/*
cypress/e2e/2-advanced-examples/*
```

Reply:
Resolved!